DigitalOffice/Services/ProjectService/Validators/NewProjectValidator.cs

```diff
@@ -8,12 +8,12 @@ public class NewProjectValidator : AbstractValidator<NewProjectRequest>
         public NewProjectValidator()
         {
             RuleFor(project => project.Name)
-                .NotEmpty()
-                .WithMessage("Project name is empty.")
-                .MaximumLength(80)
-                .WithMessage("Project name too long.")
-                .Matches("^[A-Z 0-9][A-Z a-z 0-9]+$")
-                .WithMessage("Incorrect project name.");
+                    .NotEmpty()
+                    .WithMessage("Project must have a name.")
+                    .MaximumLength(80)
+                    .WithMessage("Project name too long.")
+                    .Matches("^[A-Z 0-9][A-Z a-z 0-9]+$")
+                    .WithMessage("Incorrect project name.");
```

Comment:
Why?

Reply:
Why a lot of questions are addressed to me like I've written this code?
This is not mine. Look at the changes: I just fixed grammar even though I shouldnt have

Next time I'll touch nothing else except my code because no one seems to differentiate. My task has long been done, but this pr will not ever close. Jesus

Reply:
Removed this